lib/ex_doc/markdown/hoedown.ex

```diff
@@ -12,10 +12,11 @@ defmodule ExDoc.Markdown.Hoedown do
 
   """
   def to_html(text, opts \\ []) do
-    Markdown.to_html(text,
-      autolink: Keyword.get(opts, :autolink, true),
-      fenced_code: Keyword.get(opts, :fenced_code, true),
-      tables: Keyword.get(opts, :tables, true))
+    text
+    |> Markdown.to_html(
+         autolink: Keyword.get(opts, :autolink, true),
+         fenced_code: Keyword.get(opts, :fenced_code, true),
+         tables: Keyword.get(opts, :tables, true))
```

Comment:
particularly in this case, i would be more inclined to leave it as it was before, because it's only two blocks and the first one has so many options, I think it's more readable  as it was. but it doens't really matter to be honest.